RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -811,6 +811,34 @@ string freeRunCombatString(monster enemy, location loc, boolean inCombat)
 	string pre_banish = freeRunCombatStringPreBanish(enemy, loc, inCombat);
 	if (pre_banish != "") return pre_banish;
 
+	//Standard free-runs
+	if (!inAftercore() && have_effect($effect[Everything Looks Green]) == 0)
+	{
+		if(auto_haveSpringShoes())
+		{
+			if(!inCombat)
+			{
+				autoEquip($item[spring shoes]);
+				return "runaway item " + $item[spring shoes];
+			}
+			else
+			{
+				if(have_equipped($item[spring shoes]))
```

Comment:
Check can use for spring away instead of if shoes are equipped. Maybe there will be a future path or restriction so even with shoes on, can't use skill

Reply:
Done